docs/notifications.json

```diff
@@ -1,15 +1,18 @@
 [
-  {
-    "id": 27,
-    "text": "You can <a style=\"color: inherit;\" target=\"_blank\" rel=\"noopener\" href=\"https://twitter.com/MaterialUI\">follow us on Twitter</a> to receive exclusive tips and updates about Material-UI and the React ecosystem."
```

Comment:
I believe this message has been driving most of our new followers on Twitter. Should we keep it?

Reply:
- The message was introduced In October 2018: https://github.com/mui-org/material-ui/commit/eaa00523987075ee31d1261df6b38f33962986f5
- We went from an average of 70 new followers/month to 300 since then.

Reply:
> We went from an average of 70 new followers/month to 300 since then.

What did we get from the twitter followers? More poll participation, comments, issue participation in linked issues?

Reply:
The pros I can think of Twitter followers:

- Reach our users even when they are not directly using the product (polls, discussions)
- Reach the React community at large, when our users' retweet

Reply:
> Should we keep it?

My logic for removing it was that it isn't news or an announcement, and we already have the twitter followers button (and footer link) on the home page. A better use might be, for example, to announce Twitter polls here, and in the poll tweet put "Follow @MaterialUI for the final results of this poll".

Reply:
I would be careful with the operational cost of using multiple channels for the same use case, but for important news, like the survey, it would definitely definitely be interesting. Removing this message could be interesting to measure the impact.

Reply:
Okay, let's try it and see. If Twitter follower growth tanks, we'll have to reevaluate what the best method to promote the twitter account is.

Reply:
> The pros I can think of Twitter followers:

Sure but this is just a hypothesis. Did the participation in polls increase?

What do we get in retweets? Did you see download/funding increase?

As far as I can tell we're not monetizing the twitter account, right? It's only a promotional channel so we should see the effects on that in download/funding numbers.